src/components/Resources/useResources.tsx

```diff
@@ -53,16 +54,24 @@ const formatSmallUSD = (value: number, locale: string): string =>
   new Intl.NumberFormat(locale, {
     style: "currency",
     currency: "USD",
-    notation: "compact",
-    minimumSignificantDigits: 2,
-    maximumSignificantDigits: 2,
```

Comment:
Simplify the price formatting, so that two decimal places are always shown, no matter the number of significant figures. (i.e. `$xx.xx` versus `$x.xx`)